Feature/start home work

Aleksandr-Anokhin opened the pull request:
Параметр ems: этим свойством мы задаем размер EditText в буквах М (ems, maxEms, minEms). т.е. задавая ems мы сообщаем параметру EditText что бы он был шириной как какое-то количество букв М: android:ems="15"

p.s.  когда начал делать пункт 2 д.з. создал еще один макет в этом же проекте, но в эмуляторе он не отображался, поэтому всё упаковал в один макет.

Aleksandr-Anokhin committed:
Добавил основные элементы

Aleksandr-Anokhin committed:
Добавил основные элементы;
Поменял цвет фона, цвет Hello World, цвет ChekBox и цвет шрифта Off.

primus-anonymous commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -1,18 +1,79 @@
+    <EditText
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:inputType="textPassword"/>
```
Надо бы форматнуть разметку, здесь к примеру нет пробела перед />

Aleksandr-Anokhin committed:
Добавил основные элементы;
Поменял цвет фона, цвет Hello World, цвет ChekBox и цвет шрифта Off.
Выполнил команду Code->Reformat Code

Aleksandr-Anokhin commented:
Подредактировал ко с помощью команды Code->Reformat Code
